package.json

```diff
@@ -93,25 +92,23 @@
     "@edx/browserslist-config": "1.5.0",
     "@edx/typescript-config": "1.1.0",
     "@faker-js/faker": "^7.6.0",
-    "@openedx/frontend-build": "14.2.2",
-    "@testing-library/dom": "9.3.4",
+    "@openedx/frontend-build": "14.5.0",
+    "@testing-library/dom": "10.4.0",
     "@testing-library/jest-dom": "5.17.0",
-    "@testing-library/react": "^11.2.7",
+    "@testing-library/react": "^14.3.1",
```

Comment:
[suggestion] similar to other React 18 upgrade PRs, I might recommend upgrading `@testing-library/react` all the way to latest v16 and `@testing-library/user-event` to latest v14, and mitigating any breaking changes. Rationale: if we're upgrading RTL, we should ideally try to upgrade all its sub-packages to the latest, not just the minimum supported versions for React 18.